feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -141,9 +177,9 @@ fun RepairView() {
                     top = sizes.extraSize10
                 ),
                 options = optionsPermissionRadioButtons.toList(),
-                selectedOption = optionSelectedRadioButtons,
+                selectedOption = repairViewModel.repairModel.permissionToEnter,
                 onOptionSelected = {
-                    optionSelectedRadioButtons = it
+                    repairViewModel.getPermissionToEnterFromView(it)
```

Comment:
we need to use hight order function here and send the event to the top

Reply:
Modified! Thank you for your help